src/systems/ec/specs/components/footer/docs/usage.md

```diff
@@ -1 +1,31 @@
-Do's and don'ts
+The footer appears at the bottom of all  ec.europa.eu pages.
+
+It is divided into 2 functional areas:
+* the main content area
+* the secondary content area
+
+![Image](http://inno-ecl.s3.amazonaws.com/media/images/EC/Footer%20EC/Footer_Corporate.png)
```

Comment:
`http` -> `https`

Reply:
Ok, thanks. I will make all the new link https from now on. I have another question. Can we use SVG in some case instead of .png or .jpg? Could be convenient in some cases.

Reply:
Yes, you can use SVG. It's often an even better option actually 👍

Reply:
That's great. Makes things much easier for me!